app/assets/stylesheets/widgets/rss-feed-display.scss

```diff
@@ -0,0 +1,123 @@
+.rss-feed-display-widget {
+  margin: 2rem 0;
+  .text--link {
+    border-bottom: 2px solid currentColor;
+  }
+
+  &__title {
+
```

Comment:
should we excommunicate this guy

Reply:
@nicholasnadel good catch,  just removed it